src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -425,21 +421,23 @@
  * </ul>
  * whereby the signed-ness of the source type corresponds to the signed-ness of the promoted type. The complete process
  * of default argument promotion is described in the C specification. In effect these promotions place limits on the
- * specialized form of a variadic function, as the variadic parameters of the specialized form will always have a promoted
- * type.
+ * types that can be used to replace the {@code ...}, as the variadic parameters of the specialized form will always
```

Comment:
specialized form -> specialized function? (this was there even before, but I'm not sure "form" is too clear given previous terminology)

Reply:
Note that we use 'specialized form' in the following paragraph as well. I think it's more accurate than specialized function though, since it's the 'specialized form' of a variadic function, rather than a separate specialized function.

Reply:
Rephrased this as "as the variadic parameters of the specialized form of a variadic function will always have a promoted type."